component-resolver-core/src/main/groovy/org.octopusden/octopus/escrow/configuration/loader/ComponentRegistryInfo.groovy

```diff
@@ -35,6 +35,8 @@ class ComponentRegistryInfo {
 
     private boolean fromClassPath = false;
 
+    private final String validationExcludedComponentsFileName = "excludedComponents.txt"
```

Comment:
As named, excludedComponents.txt reads like a global exclusions list for all validations. In our case this file is used for a specific validation only.

One of the approached can be used

1) Rename the file to reflect the rule it belongs to, e.g.
excluded-components-<validationId>.txt
validation-exclusions/<validationId>.txt

2) Keep the generic filename but store a structured mapping of validation → components
Examples:
```
<validationId>:
  - compA
  - compB
```

```
{ "<validationId>": ["compA","compB"] }
```